src/PeerDigest.h

```diff
@@ -79,9 +79,9 @@ class PeerDigest
     PeerDigest(CachePeer *);
     ~PeerDigest();
 
-    CachePeer *peer = nullptr;          /**< pointer back to peer structure, argh */
+    CbcPointer<CachePeer> peer;           /**< pointer back to peer structure, argh */
```

Comment:
Same alignment request here. And we can use simpler/shorter (and more common in Squid) Doxygen syntax:

```suggestion
    CbcPointer<CachePeer> peer; ///< pointer back to peer structure, argh
```